forge-gui/res/cardsfolder/upcoming/annie_flash_the_veteran.txt

```diff
@@ -8,7 +8,8 @@ SVar:TrigChangeZone:DB$ ChangeZone | Origin$ Graveyard | Destination$ Battlefiel
 T:Mode$ Taps | ValidCard$ Card.Self | Execute$ ExileTwo | TriggerDescription$ Whenever NICKNAME becomes tapped, exile the top two cards of your library. You may play those cards this turn.
 SVar:ExileTwo:DB$ Dig | Defined$ You | DigNum$ 2 | ChangeNum$ All | DestinationZone$ Exile | RememberChanged$ True | SubAbility$ DBEffect
 SVar:DBEffect:DB$ Effect | RememberObjects$ RememberedCard | StaticAbilities$ STPlay | SubAbility$ DBCleanup | ForgetOnMoved$ Exile | Duration$ EndOfTurn
-SVar:STPlay:Mode$ Continuous | MayPlay$ True | Affected$ Card.IsRemembered | AffectedZone$ Exile | Description$ Until the end of your next turn, you may play the exiled cards.
+SVar:STPlay:Mode$ Continuous | MayPlay$ True | Affected$ Card.IsRemembered | AffectedZone$ Exile | Description$ You may play the exiled cards until the end of turn.
```

Comment:
Why not just stick to the Oracle?

Reply:
I like to err on the side of the text being a bit clearer. 
Incidentally I had a wider look at the state of things for these particular lines of text and a cleanup is in order.

Reply:
Always plenty to clean up 😅

Reply:
Well then, split the difference into something shorter and with precedent in existing scripts (Korvold and the Noble Thief), and cleaned up where relevant.

Reply:
Nice work – some of those were really bad